### PR TITLE
chore: Improve French translation accuracy

### PR DESCRIPTION
**Pull request by lucasjoliveau**
## What does this PR do?

This PR improves accuracy in the French translations. It aims to enhance the user experience for French-speaking users by providing more accurate and reliable translations.

## Requirement/Documentation

No additional documentation or requirements were necessary for these changes.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Testing should focus on reviewing the corrected translations to ensure they accurately reflect the intended meanings. Ideally, a native French speaker should verify the translations for accuracy and context.

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by lucasjoliveau** [comment]
I've noticed during the process of improving the French translations that there are several instances within the application where text is hardcoded and not included in i18n strings. Making these texts part of the i18n strings would greatly enhance the application's flexibility and accessibility for users in different languages.

Would it be possible to consider this improvement in future updates?

File: apps/web/public/static/locales/fr/common.json [hunk 1]
```diff
@@ -2218,15 +2218,15 @@
   "not_enough_available_seats_error": "La réservation n'a pas assez de places disponibles",
   "user_redirect_title": "{{username}} est actuellement en absence de courte durée.",
   "user_redirect_description": "Pendant ce temps, {{profile.username}} sera responsable de tous les rendez-vous programmés de la part de {{username}}.",
-  "out_of_office": "Pas au bureau",
+  "out_of_office": "En dehors du bureau",
```
Comment (ThyMinimalDev) [inline apps/web/public/static/locales/fr/common.json:2221]: I feel like in french "absent" for out of office it's the most common way to say it

Comment (lucasjoliveau) [reply apps/web/public/static/locales/fr/common.json:2221]: Thank for your feedback @ThyMinimalDev 

As a French speaker, "absent" is commonly used similarly to "unavailable," but "out of the office" directly translates to "En dehors du bureau," specifying absence from the workplace.

Comment (ThyMinimalDev) [reply apps/web/public/static/locales/fr/common.json:2221]: Hey there, not sure about this, I am a french speaker myself too and I have never heard or seen this expression, being "out-of-office / OOO" does not directly translate to being "outside of the office" it means you are unavailable for meetings / work in general. so in the specific context of a calendar / booking system this being translated as "Absent du bureau" makes more sense.

can you make the change please and I'll approve your pr 👍 

Thanks

Comment (Jojo14234) [reply apps/web/public/static/locales/fr/common.json:2221]: Hi, I'm a French speaker as well and would have translated this simply to "Absences" considering where it is being used in the UI. Unlike OOO in English, "Absent du bureau" isn't a very common thing to read or hear so I think "Absences" better indicates what that part of the settings does. Cheers

**Comment by keithwillcode** [comment]
I went ahead and made the changes in #14718. I wasn't able to directly update this PR.

**Pull request closed without merging**